src/jdk.incubator.code/share/classes/jdk/incubator/code/bytecode/BytecodeLift.java

```diff
@@ -474,6 +475,11 @@ yield op(JavaOp.invoke(JavaOp.InvokeOp.InvokeKind.SUPER, false,
                         JavaOp.LambdaOp.Builder lambda = JavaOp.lambda(currentBlock.parentBody(),
                                                                        lambdaFunc,
                                                                        JavaType.type(inst.typeSymbol().returnType()));
+                        // if FLAG_QUOTABLE is set, the lambda is quotable
+                        if (inst.bootstrapArgs().get(inst.bootstrapArgs().size() - 3) instanceof Integer a
```

Comment:
IIUC the flags argument should always be the fourth bootstrap argument? See [here](https://github.com/openjdk/babylon/blob/9ca6a4243794b974562c43ed6072dc0575c418ba/src/java.base/share/classes/java/lang/invoke/LambdaMetafactory.java#L510)